README.md

```diff
@@ -180,6 +180,16 @@ This is from [nekop's japanese article](https://nekop.hatenablog.com/entry/2018/
 
 zookeeper gem doesn't work on Debian 10, so kafka image doesn't include zookeeper gem.
 
+### Windows k8s daemonset doesn't support in this repository
```

Comment:
Maybe rephrase this to: "Windows k8s daemonset not supported in this repository"

Reply:
Passive voice is more natural in title?

Reply:
Ah, I got it. Will do.